app/src/main/java/org/sopt/at/HomeActivity.kt

```diff
@@ -0,0 +1,254 @@
+                    }
+                }
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalFoundationApi::class)
+@Composable
+fun HomeScreen(modifier: Modifier = Modifier) {
+    LazyColumn(
+        modifier = modifier
+            .fillMaxSize(),
+        verticalArrangement = Arrangement.spacedBy(20.dp)
+    ) {
+        item { TvingTopBar() }
+        stickyHeader {
+            Surface(color = Color.Black) {
+                CategoryTab()
+            }
+        }
+        item { BannerSection() }
+
+        // 필수 섹션
+        item { SectionWithRow(title = "오늘의 티빙 TOP 20") }
+        item { SectionWithRow(title = "지금 방영 중인 콘텐츠") }
+    }
+}
+
+@Composable
+fun CategoryTab() {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(vertical = 10.dp),
+        horizontalArrangement = Arrangement.SpaceEvenly
+    ) {
+        val categories = listOf("드라마", "예능", "영화", "스포츠", "애니", "뉴스")
+        var selectedIndex by remember { mutableStateOf(0) }
+
+        categories.forEachIndexed { index, title ->
+            Text(
+                text = title,
+                modifier = Modifier
+                    .clickable { selectedIndex = index },
+                fontWeight = if (selectedIndex == index) FontWeight.Bold else FontWeight.Normal,
+                color = if (selectedIndex == index) Color.White else Color.Gray
+            )
+        }
+    }
+}
+
+@Composable
+fun BannerSection() {
+    Box(modifier = Modifier.fillMaxWidth()) {
+        Image(
+            painter = painterResource(id = R.drawable.img_20),
+            contentDescription = "Mock Banner",
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(500.dp),
+            contentScale = ContentScale.Crop
+        )
+    }
+}
+
+@Composable
+fun SectionWithRow(title: String) {
+    val context = LocalContext.current
+    val imageList = if (title == "오늘의 티빙 TOP 20") {
+        emptyList()
+    } else {
+        listOf(
+            R.drawable.img_01, R.drawable.img_02, R.drawable.img_03, R.drawable.img_04, R.drawable.img_05,
+            R.drawable.img_06, R.drawable.img_07, R.drawable.img_08, R.drawable.img_09, R.drawable.img_10,
+            R.drawable.img_11, R.drawable.img_12, R.drawable.img_13, R.drawable.img_14, R.drawable.img_15,
+            R.drawable.img_16, R.drawable.img_17, R.drawable.img_18, R.drawable.img_19, R.drawable.img_20
+        ).shuffled().take(10)  //랜덤하게 10개 보여주기
+    }
+    Column(modifier = Modifier.padding(start = 16.dp, bottom = 16.dp)) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(text = title, fontSize = 18.sp, fontWeight = FontWeight.Bold, color = Color.White)
+            Text(text = "더보기", modifier = Modifier.padding(end = 16.dp), color = Color.White)
+        }
```

Comment:
```suggestion
Row(
            modifier = Modifier.fillMaxWidth(),
            horizontalArrangement = Arrangement.SpaceBetween,
            verticalAlignment = Alignment.CenterVertically
        ) {
            Text(
                text = title,
                fontSize = 18.sp,
                fontWeight = FontWeight.Bold, color = Color.White
            )
            Text(
                text = "더보기",
                modifier = Modifier.padding(end = 16.dp),
                color = Color.White
            )
        }
```

이러면 더 가독성이 좋을 것 같아요.
그리고 패딩값이 여기저기 섞여있는데 더 추적하기 좋게 패딩을 주면 어떨까요?
Text("더보기")가 end에 16을 가지는것보단 부모에서 적용하는게 더 좋은 것 같아요.